app/common/src/popup/BarPopup.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+
+import * as React from 'react';
+import { View, StyleSheet } from 'react-native';
+
+import Popup from './Popup';
+import Color from '../Color';
+import LinkButton from '../buttons/LinkButton';
+
+type Props = {|
+  buttonTitle: string,
+  children: React.Node,
+  isVisible: boolean,
+  onClose: Function,
+  onSave: Function,
+|};
+
+export default class BarPopup extends React.Component<Props> {
```

Comment:
I have no idea what is "bar" popup and "button" popup just from looking at these components. Please try to add doc comments to make it clear immediately without looking for usage and trying to figure it out from code (or try to find better naming to help your colleagues)... 😊 

Something like: "this popup has a small bar at the top with cancel and ... button" or "this is an ordinary popup with confirmation button"...